README.md

```diff
@@ -359,6 +359,7 @@ Table of Contents
   * [Directus](https://directus.io) — Headless CMS. A completely free and open-source platform for managing assets and database content on-prem or in the Cloud. No limitations or paywalls.
   * [Forestry.io/](https://forestry.io/) — Headless CMS. Give your editors the power of Git. Create and edit Markdown-based content with ease. Comes with three free sites that includes 3 editors, Instant Previews. Integrates with blogs hosted on Netlify/GitHubpages/ elsewhere
   * [kontent.ai](https://www.kontent.ai) - A Content-as-a-Service platform that gives you all the headless CMS benefits while empowering marketers at the same time. Developer plan provides 2 users with unlimited projects with 2 environments for each, 500 content items, 2 languages with Delivery and Management API, and Custom elements support. Larger plans available to meet your needs.
+  * [Payload CMS](https://payloadcms.com) — Headless and self-hosted, JavaScript platform without limits. Build your content structures in code as simple configuration and develop beyond. Fully featured Personal plan includes no-caps GraphQL &amp; REST APIs, authentication, localization, file uploads and more for rich content experiences and application development.
```

Comment:
this seems nice, but this list is for software delivered As a Service. Not for self hosting, sorry.